FiveCalls/FiveCalls/AppState.swift

```diff
@@ -24,7 +24,11 @@ class AppState: ObservableObject, ReduxState {
         }
     }
     @Published var donateOn = false
-    @Published var issues: [Issue] = []
+    @Published var issues: [Issue] = [] {
+        didSet {
+            issueLoadedCallback?()
```

Comment:
rather than require this temporary callback, couldn't we use middleware to do this only the first time issues are set?